packages/wallets/src/features/cashier/modules/FiatOnRamp/__tests__/FiatOnRamp.spec.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+import { APIProvider } from '@deriv/api';
+import { fireEvent, render, screen } from '@testing-library/react';
+import FiatOnRamp from '../FiatOnRamp';
+
+jest.mock('../components/FiatOnRampDisclaimer', () => ({
+    ...jest.requireActual('../components/FiatOnRampDisclaimer'),
+}));
+
+jest.mock('../components/FiatOnRampProviderCard', () => ({
+    ...jest.requireActual('../components/FiatOnRampProviderCard'),
+}));
+
+jest.mock('../constants', () => ({
+    ...jest.requireActual('../constants'),
+}));
```

Comment:
I dont think you need to mock these if you are not gonna mock something inside of that component / file